cmd/probe/probe.go

```diff
@@ -27,7 +27,7 @@ const (
 )
 
 // RegisteredProbes contains register function of probes which we want to register
-var RegisteredProbes = []func(){smartProbeRegister, udevProbeRegister}
+var RegisteredProbes = []func(){smartProbeRegister, udevProbeRegister, ebsProbeRegister}
```

Comment:
break this line into multiple lines [like this](https://github.com/openebs/node-disk-manager/blob/367dcc004c55b84f7de442f9631a5f027f488fcd/cmd/filter/filter.go#L29)